conf/messages.en

```diff
@@ -192,8 +192,8 @@ addTradingName.title = You have added {0} UK trading names
 addTradingName.heading = You have added {0} UK trading names
 addTradingName.maximumReached = You have added the maximum number of trading names.
 addTradingName.error.required = Select yes if you need to add another UK trading name
-addTradingName.change.hidden = Change {0}
-addTradingName.remove.hidden = Remove {0}
+addTradingName.change.hidden = {0}
+addTradingName.remove.hidden = {0}
```

Comment:
Shouldn't we use the value directly if there's no text?

Reply:
I see, it's being used in `addToList.scala.html`. We could update/rename the `changePrefix` and `removePrefix` parameters to be Long

Reply:
Looks like IOSS also uses `= {0}` - feel free to ignore my comment then